src/profile-logic/processed-profile-versioning.js

```diff
@@ -1807,5 +1807,200 @@ const _upgraders = {
       }
     }
   },
+  [38]: (profile) => {
+    // The frame table no longer contains return addresses, it now contains
+    // "nudged" return addresses, i.e. return address minus one byte.
+    // See nudgeReturnAddresses for more details.
+    // The code from nudgeReturnAddresses is duplicated below because this
+    // upgrader needs to stay unaffected by any profile format changes after
+    // version 38.
+    //
+    // This upgrader is needed so that clicking "Re-symbolicate" on old profiles
+    // will obtain correct line numbers and inline frames (once implemented),
```

Comment:
We can do that once the new symbol server is in production.

I left out one crucial piece of information here: Existing profiles don't actually contain incorrect line numbers, because existing profiles don't contain **any** line numbers! That's because the symbol server that's in production at the moment does not return line information.
The only way I got these demonstration profiles was by using the `symbolServer` URL param to point to the staging symbol server.

So, to get any file names and line numbers on old profiles, users will always have to re-symbolicate first.